tracer/test/Directory.Build.props

```diff
@@ -10,10 +10,6 @@
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
     <!-- Stop NuGet from complaining about vulnerable packages -->
     <NuGetAudit>false</NuGetAudit>
-
-    <!-- WebClient is obsolete -->
```

Comment:
Do we still want the `<NoWarn>SYSLIB0014</NoWarn>` ?

Reply:
I don't know 😅 We don't seem to need it so 🤷‍♂️